packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -146,53 +303,123 @@ class FlutterErrorDetails {
     return longMessage;
   }
 
+  Diagnosticable _exceptionToDiagnosticable() {
+    if (exception is FlutterError) {
+      return exception;
+    }
+    if (exception is AssertionError && exception.message is FlutterError) {
+      return exception.message;
+    }
+    return null;
+  }
+
+  /// Returns a short (one line) description of the problem that was detected.
+  ///
+  /// If the exception contains an [ErrorSummary] that summary is used otherwise
```

Comment:
not 100% sure: Should there be some kind of punctuation before otherwise?

Reply:
added a comma